mango-core/pom.xml

```diff
@@ -176,5 +176,30 @@
       <artifactId>hadoop-bam</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <version>3.0.0-beta-3</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java-util</artifactId>
+      <version>3.0.0-beta-3</version>
+    </dependency>
+    <dependency>
+      <groupId>org.ga4gh</groupId>
+      <artifactId>ga4gh-schemas</artifactId>
+      <version>0.6.0a10</version>
+    </dependency>
+    <dependency>
+    <groupId>org.bdgenomics.convert</groupId>
+    <artifactId>convert-ga4gh</artifactId>
+    <version>0.2.0-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>org.bdgenomics.convert</groupId>
+      <artifactId>convert</artifactId>
+      <version>0.2.0-SNAPSHOT</version>
```

Comment:
this is dependent on a new release of bdg-convert.
https://github.com/bigdatagenomics/bdg-convert/issues/36